### ISSUE TITLE
Not working on iPhone 7 

### ISSUE THREAD

**Issue by Cexdexbbl** [checkra1n/BugTracker#1819]
**Tell us about your setup:**
1. What iDevice are you using?
     iPhone 7  
2. On what version of iOS is it?
     14.0.1, 14.2 
3. What version of checkra1n are you using?
   0.12.0
4. What is your host system (OS version? Hackintosh? VM? etc.)?
    macOS Catalina 
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
     Apple cable 
**What are the steps to reproduce the issue?**
1. Follow instructions that checkra1n says to do 
2. 
3.
...

**What do you expect, and what is happening instead?**
To be able to jailbreak. No checkra1n app appears when "All done" is displayed 

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**

yes, safe mode seems not to be working. 

**Any other info, error logs, screenshots, ...?**
when I first downloaded the checkra1n it didn't let me JB said that I had an "error -20" I tried safe mode but it didn't work. I kept trying but it didn't work. I downloaded checekra1n again but this time I'm able to get to "All done" step but nothing appears. Also, I tried another computer it didn't work.

**Comment by TezIa** [comment]
Same for me. But i am also getting -77 usb error but i dont think thats the main problem. I do the regular checkra1n suff just as usual but the Loader doesn't show up.

**Comment by Cexdexbbl** [comment]
> Same for me. But i am also getting -77 usb error but i dont think thats the main problem. I do the regular checkra1n suff just as usual but the Loader doesn't show up.

yeah I'm kinda disappointed about this. I've never had a problem using checkrain before. hopefully, they're able to fix.

**Comment by ivanjermakov** [comment]
The same issue, but at first I was able to jailbreak on iOS 14.2 yesterday.
No checkra1n loader show up and Cydia and Installer not working. But for some reason, NewTerm and Filza work fine, even under root.

**Comment by raspberryenvoie** [comment]
I'm having the same issue as you

**Comment by raspberryenvoie** [comment]
I didn't work on Linux. I've just tried on macOS and it worked!

**Comment by raspberryenvoie** [comment]
It didn't work on Linux. I've just tried on macOS and it's working!

**Comment by Cexdexbbl** [comment]
> The same issue, but at first I was able to jailbreak on iOS 14.2 yesterday.
> No checkra1n loader show up and Cydia and Installer not working. But for some reason, NewTerm and Filza work fine, even under root.

I believe it's because you are Jailbroken but Cydia doesn't appear for a reason.

**Comment by Cexdexbbl** [comment]
**Potential Fix** 

So I was trying to figure out why checkrain bootloader wouldn't show up. I did research, which let me to install **Sileo**. I noticed that when I updated some files from Sileo and it re-springed my phone **the checkrain loaded appeared** and I was able to install cydia etc... from checkrain bootloader but if you install Sileo don't install Cydia it causes problems. 
I used this link to install Sileo. https://odysseyra1n.com/
